lang/en/docs/installing-dependencies.md

```diff
@@ -5,13 +5,15 @@ layout: guide
 additional_reading_tags: ["dependencies", "package-json", "yarn-lock", "cli-install"]
 ---
 
-If you have just checked out a package from [version control](./version-control), you will need to install those dependencies.
+{% include vars.html %}
```

Comment:
Yup, not necessary, just the same format as the other md files in this folder. As said in my previous comment, I can remove it if you want.